bin/build_helpers.sh

```diff
@@ -14,9 +14,15 @@ check_rust() {
   fi
 
   if ! which rustup > /dev/null; then
-    echo 'error: rustup not found in PATH' >&2
-    echo 'note: Rust can be installed from https://rustup.rs/' >&2
-    exit 1
+    if ! which cargo > /dev/null; then
+      echo 'error: cargo not found in PATH; do you have Rust installed?' >&2
+      echo 'note: Rust can be installed from https://rustup.rs/' >&2
```

Comment:
I think it can help to specify that this is not just _the_ or _a_ Rust installation, but that we recommend rustup in particular?

       echo 'note: We recommend installing Rust via rustup from https://rustup.rs/' >&2